### PR TITLE
Footer Added

### PR DESCRIPTION
**Pull request by chirag08**
Added and merged footer code and Styling files .

### CODE REVIEW

**Comment by CuriousLearner** [comment]
@chirag08 When You're done, tell us here, but before that sqash your commits :+1: 

Also, make title of your pull request more descriptive from next time.

**Comment by chirag08** [comment]
Done All Styling on footer to make it responsive for Mobile and tablets as suggested by @Slayslot  and Resolved merge conflicts

**Comment by chirag08** [comment]
@CuriousLearner @Slayslot kindly Review it for merging

**Comment by CuriousLearner** [comment]
@Slayslot Look into this, and if this is fine. Please consider squashing the commits first and then merge.

**Comment by MozillaDelhi** [comment]
@Slayslot  Please review the pull request ASAP.

**Comment by handasolo** [comment]
@chirag08 I recently pushed 2 commits, so while I review please make sure this PR is up to date with develop. Also, consider [squashing your commits](http://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git) as @CuriousLearner has suggested.

**Comment by chirag08** [comment]
ohke let me update my PR

**Comment by CuriousLearner** [comment]
Make your commit message something similar to "Footer added." after squashing. That would be good. :+1:

File: src/client/index.html [hunk 1]
```diff
@@ -196,6 +275,6 @@ <h1 id="ff-download-btn-h" class="col-sm-12">
       <script src="https://mozorg.cdn.mozilla.net/tabzilla/tabzilla.js"></script>
       <!-- Custom Script -->
       <script src="js/script.js"></script>
-    </div>
```
Comment (askprateek) [inline src/client/index.html:199]: @chirag08 Check this out

File: src/client/index.html [hunk 2]
```diff
@@ -178,24 +178,98 @@ <h1 id="ff-download-btn-h" class="col-sm-12">
             </div>
           </div>
           <span id="success" class="col-xs-12 col-sm-12 col-md-6 col-lg-6 form-group" style="color:white;"></span>
-          <input type="submit" name="send" id="send" value="Send" class="btn btn-info pull-right">
+          <input type="submit" name="send" id="send" value="Send" class="btn btn-info pull-right send-button" style="margin-bottom:15px; ">
```
Comment (handasolo) [inline src/client/index.html]: Can you push the inline-style to the css file? Must've forgot this.

Comment (chirag08) [inline src/client/index.html]: inline style  in 181 line only ? @Slayslot

Comment (handasolo) [inline src/client/index.html]: Yeah, that's the one you put regarding footer.

**Comment by chirag08** [comment]
Pushed that minor change

**Comment by handasolo** [comment]
Thank you.

**Comment by handasolo** [comment]
Thank you for your patience with this one.

**Pull request merged**